docs/testing/playwright/03-api.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Playwright Adapter API
+sidebar_label: API
+---
+
+## `createStencilPlaywrightConfig` Function
+
+**Signature:** `createStencilPlaywrightConfig(overrides?: CreateStencilPlaywrightConfigOptions): Promise<PlaywrightTestConfig>`
+
+Returns a [Playwright test configuration](https://playwright.dev/docs/test-configuration#introduction).
+
+`overrides`, as the name implies, will overwrite the default configuration value(s) if supplied. These values can include any valid Playwright config option as well
+as some options to override specific values in the config options related to the Stencil integration:
+
+- `webServerCommand`: This can be specified to change the command that will run to start the Stencil dev server. Defaults to `npm start -- --no-open`.
+- `webServerTimeout`: This can be specified to change the amount of time Playwright will wait for the dev server to start. Defaults to 60 seconds.
```

Comment:
Can we clarify the unit of this configuration options is millis? I think it can get a little confusing since we say it defaults to 60 seconds, then use millis in the example.
```suggestion
- `webServerTimeout`: This can be specified to change the amount of time Playwright will wait for the dev server to start in milliseconds. Defaults to 60000 (60 seconds).
```